Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -47,97 +52,157 @@ public void TestPutCommand(string compressionType)
             string createTable = $"create or replace table {TEST_TEMP_TABLE_NAME} ({COL1} STRING," +
             $"{COL2} STRING," +
             $"{COL3} STRING)";
-            string copyInto = $"COPY INTO {TEST_TEMP_TABLE_NAME}";
+            string createStage = $"create or replace stage {TEST_TEMP_STAGE_NAME}";
```

Comment:
For stage creating, please refer to https://docs.snowflake.com/en/sql-reference/sql/create-stage.html, include tests cover both options: ENCRYPTION = (TYPE = 'SNOWFLAKE_FULL' | TYPE = 'SNOWFLAKE_SSE') . Since sometime dot net code might have to work on already created stages.